src/main.ts

```diff
@@ -68,6 +79,7 @@ async function analyzeCode(
     if (file.to === "/dev/null") continue; // Ignore deleted files
     for (const chunk of file.chunks) {
       const prompt = createPrompt(file, chunk, prDetails);
+      console.log(">>> prompt", prompt)
```

Comment:
Remove the `console.log` statement used for debugging purposes. It should not be present in the production code.